### PR TITLE
212 feature add confirmation mail after register

### LINKED ISSUE #212: [FEATURE]: Add Confirmation mail after register
### Description

After registration, the user shall receive an email with a confirmation link.
After successfully registering with the link, the user may have access to the Area App.

### To do

Implement with Firebase a way to send a confirmation mail.

### Priority

High

### Project parts

- [X] Server
- [X] Application
- [X] Web
- [ ] Automation

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by osvegn**
# New feature
Email user to validate his registration.
Ask him to validate registration before connecting.

### CODE REVIEW

File: Web/src/AuthPage.js [hunk 1]
```diff
@@ -126,8 +126,13 @@ function AuthPage(props) {
                                     setIsBadPassword(false);
                                     props.userInformation.id = data.userUid;
                                     props.userInformation.mail = email;
-                                    addDataIntoCache("area", { mail: props.userInformation.mail, id: props.userInformation.id, password: btoa(password), ip: props.userInformation.ip });
-                                    navigate('/home');
+                                    if (endpoint === '/register') {
+                                        alert("Vous êtes bien inscrit ! Vérifier votre boite mail pour confirmer votre inscription. Vous pouvez maintenant vous connecter.")
```
Comment (JohanCDev) [inline Web/src/AuthPage.js:130]: Vérifiez

**Review by JohanCDev — CHANGES_REQUESTED** [review]

File: Web/src/AuthPage.js (hunk 1 above)
Comment (osvegn) [reply Web/src/AuthPage.js:130]: It's not an error, "verifier" is the good word

**Review by CedricCORGE — APPROVED** [review]
lgtm

**Review by JohanCDev — APPROVED** [review]
Not sure about "Vérifier" but if you are lgtm

**Pull request merged**